From f5eb5f7891907414caccd1d82c7171bdfafe7900 Mon Sep 17 00:00:00 2001
From: Adam Wujek <adam.wujek@cern.ch>
Date: Wed, 18 May 2016 09:22:55 +0200
Subject: [PATCH] kernel: remove deprecated use of defined in
 kernel/timeconst.pl

Fix the warning (for perl < 5.21.x) whihch becomes an error
(for perl >= 5.21.x):

defined(@array) is deprecated at kernel/timeconst.pl line 373.
  (Maybe you should just omit the defined()?)

Signed-off-by: Adam Wujek <adam.wujek@cern.ch>
---
 kernel/timeconst.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/timeconst.pl b/kernel/timeconst.pl
index eb51d76..0461239 100644
--- a/kernel/timeconst.pl
+++ b/kernel/timeconst.pl
@@ -370,7 +370,7 @@ if ($hz eq '--can') {
 	}
 
 	@val = @{$canned_values{$hz}};
-	if (!defined(@val)) {
+	if (!@val) {
 		@val = compute_values($hz);
 	}
 	output($hz, @val);
-- 
1.9.1

